org.civicrm.reqangular/test/test-main.js

```diff
@@ -38,6 +38,15 @@ require.config({
         'common/angularUiRouter': {
             deps: ['common/angular']
         },
+        'common/angularUiRouter': {
+            deps: ['common/angular']
```

Comment:
I've just conformed to the current file indentation, in order to make it easier to spot what's been changed. We can fix those indentations later.
Anyway, if you think it's ok to fix them now, I can do it ;-)